apps/site/pages/fr/about/security-reporting.mdx

```diff
@@ -80,4 +80,4 @@ le référentiel [nodejs/security-wg](https://github.com/nodejs/security-wg).
   />
 </a>
 
-Le [badge des meilleures pratiques](https://github.com/coreinfrastructure/best-practices-badge) de l'Open Source Security Foundation (OpenSSF) est un moyen pour les projets de logiciels libres et open source (FLOSS) de montrer qu'ils suivent les meilleures pratiques. Les projets peuvent volontairement auto-certifier la manière dont ils suivent chaque meilleure pratique. Les utilisateurs du badge peuvent rapidement déterminer quels sont les projets FLOSS qui suivent les meilleures pratiques et qui sont donc plus susceptibles de produire des logiciels sécurisés de meilleure qualité.
+Le \[badge des meilleures pratiques] de l'Open Source Security Foundation (OpenSSF) ([https://github.com/coreinfrastructure/best-practices-badge](https://github.com/coreinfrastructure/best-practices-badge)) est un moyen pour les projets de logiciels libres et open source (FLOSS) de montrer qu'ils suivent les meilleures pratiques. Les projets peuvent volontairement auto-certifier la manière dont ils suivent chaque meilleure pratique. Les utilisateurs du badge peuvent rapidement déterminer quels sont les projets FLOSS qui suivent les meilleures pratiques et qui sont donc plus susceptibles de produire des logiciels sécurisés de meilleure qualité.
```

Comment:
Someone need to go on crowdin to fix that (I technically can but I am away from my computer)

Reply:
i think i fixed it (for next run)